web/src/components/nomination/TopicsInput.vue

```diff
@@ -1,14 +1,10 @@
 <template>
   <v-row dense>
     <v-col>
-      <v-autocomplete
```

Comment:
Oh perhaps I misunderstood your comment this morning... were you talking about changing the airtable form, or changing this? Because the autocomplete with multiple chips could work better than splitting up a string for both the UI and ease of data entry.

Reply:
Ah, I agree, autocomplete provides a better UX. The problem is that Airtable does not allow to create extra options dynamically for "multiple string select" columns.

The solution I see is to extract `Topics` into a separate table (like we're doing with `Location`) and then handle loading existing options and creating new options on the fly all on the front-end. It's some level of extra work, but maybe worth it?

Can we do it on a separate PR though? (I can revert this back to autocomplete with limited options/no creation of new options if that's better for now)

Reply:
Yeah let's solve on a separate PR. It's subtle and we don't need topics right away anyway, we can collect people through the airtable form in the interim. 💪 👩